internal/common/config.go

```diff
@@ -16,21 +16,21 @@
 
 package common
 
+import "github.com/edgexfoundry/go-mod-messaging/pkg/types"
+
 // WritableInfo ...
 type WritableInfo struct {
-	SubscribeTopic string
-	PublishTopic   string
-	LogLevel       string
+	LogLevel string
 }
 
 // ConfigurationStruct ...
 type ConfigurationStruct struct {
-	Writable   WritableInfo
-	Logging    LoggingInfo
-	Registry   RegistryInfo
-	Service    ServiceInfo
-	MessageBus MessageBusInfo
-	Binding    Binding
+	Writable            WritableInfo
+	Logging             LoggingInfo
+	Registry            RegistryInfo
+	Service             ServiceInfo
+	MessageBusConfig    types.MessageBusConfig
```

Comment:
I prefer the way this was before. I think `ConfigurationStruct.MessageBusConfig` is a form a stutter.

Reply:
Yup, agreed.